packages/gui/src/components/schemas/gradient.tsx

```diff
@@ -8,10 +9,10 @@ import { position } from './position'
 import { angle, keyword } from './primitives'
 import { DataTypeSchema } from './types'
 
-export const stringifyStops = (stops: GradientStop[], unit: string = '%') => {
+export const stringifyStops = (stops: GradientStop[], theme?: Theme, unit: string = '%') => {
```

Comment:
Not sure why theme is an arg here if it isn't being used? This a leftover from previous code?